lib/include/ultrahdr/jpegr.h

```diff
@@ -79,10 +79,14 @@ typedef struct jpegr_info_struct* jr_info_ptr;
 
 class JpegR {
  public:
-  JpegR(size_t mapDimensionScaleFactor = kMapDimensionScaleFactorDefault,
-        int mapCompressQuality = kMapCompressQualityDefault,
-        bool useMultiChannelGainMap = kUseMultiChannelGainMapDefault,
-        float gamma = kGainMapGammaDefault);
+  JpegR(
+#ifdef UHDR_ENABLE_GLES
```

Comment:
Ram,
This change causes build failure in Android.

When this header is included from a different project, UHDR_ENABLE_GLES will need to be explicitly defined in that project's Android.bp as well.

So this macro in this header file isn't a good idea. Need to figure out how to handle this.